src/format/KdbxWriter.h

```diff
@@ -52,6 +52,10 @@ class KdbxWriter
      */
     virtual bool writeDatabase(QIODevice* device, Database* db) = 0;
 
+    virtual quint32 getFormatVersion() = 0;
```

Comment:
Remove the format version and add a short descriptive doc string (pure-virtual methods should never be undocumented). For parity, you may want to add the same method to the KdbxReader class.

Reply:
@phoerious I thought about adding the same method to `KdbxReader`, but didn't do it since it doesn't have a constant version (dynamically set when reading the database). I could add it, I don't have a strong opinion for or against it.

Reply:
True. The Reader class determines the version from whatever it gets as input. It might make sense to add such a method anyway, but then we'd also need some sort of default value that's returned if nothing has been read so for. I guess we can leave it as is then.